client/js/jsonedit.js

```diff
@@ -785,9 +785,8 @@ async function jeApplyChanges() {
     jeDeltaIsOurs = true;
     await jeApplyExternalChanges(completeState);
     jeStateBeforeRaw = currentStateRaw;
-    const oldState = jeStateBefore;
     jeStateBefore = currentState;
-    await updateWidget(currentState, oldState); // in editmode.js
+    await updateWidget(currentState, jeStateBefore); // in editmode.js
     jeDeltaIsOurs = false;
   }
```

Comment:
@ArnoldSmith86 How was this variable unnecessary? (jeStateBefore is reset in the intervening statement.)

Reply:
Reverted this change; testcafe tests now pass.

Reply:
Oh. Guess it was too late yesterday.